shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -517,6 +521,39 @@ - (NSString*)accessibilityLabel {
   return label;
 }
 
+- (bool)containsPoint:(CGPoint)point {
+  // The point is in global coordinates, so use the global rect here.
+  return CGRectContainsPoint([self globalRect], point);
+}
+
+// Finds the first eligiable semantics object in hit test order.
+- (SemanticsObject*)search:(CGPoint)point {
+  if ([self children].count == 0) {
```

Comment:
I think this if branch is not needed if you handle the self containsPoint in line 546